base/linalg/ldlt.jl

```diff
@@ -20,6 +20,26 @@ convert(::Type{Factorization{T}}, F::LDLt{S,U}) where {T,S,U} = convert(LDLt{T,U
     ldltfact!(S::SymTridiagonal) -> LDLt
 
 Same as [`ldltfact`](@ref), but saves space by overwriting the input `A`, instead of creating a copy.
```

Comment:
Perhaps fix `A` -> `S` such that the signature and docstring matches.

Reply:
Perhaps I wasn't clear enough; what I meant was that in the docstring we use `A` but in the signature two rows above we call it `S`.

Reply:
You were probably plenty clear, I'm just braindead on Friday afternoon 😛